config/routes.rb

```diff
@@ -326,7 +326,9 @@
     end
 
     scope '/verify/v2' do
+      get '/' => 'verify#show'
       get '/personal_key' => 'verify#show'
+      get '/personal_key_confirm' => 'verify#show'
```

Comment:
could DRY this?
```suggestion
      get '/' => 'verify#show'
      %w[
        /
        /personal_key
        /personal_key_confirm
      ].each { |step_path| get step_path => 'verify#show' }
```

Reply:
> could DRY this?

Yeah, this idea's much better than the regex idea I'd had previously in https://github.com/18F/identity-idp/pull/6177#discussion_r845311146 .